libcomposefs/erofs_fs.h

```diff
@@ -127,11 +128,30 @@ static inline bool erofs_inode_is_data_compressed(unsigned int datamode)
 #define EROFS_CHUNK_FORMAT_ALL	\
 	(EROFS_CHUNK_FORMAT_BLKBITS_MASK | EROFS_CHUNK_FORMAT_INDEXES)
 
+/* 32-byte on-disk inode */
+#define EROFS_INODE_LAYOUT_COMPACT	0
+/* 64-byte on-disk inode */
+#define EROFS_INODE_LAYOUT_EXTENDED	1
+
 struct erofs_inode_chunk_info {
 	__le16 format;		/* chunk blkbits, etc. */
 	__le16 reserved;
 };
 
+union erofs_inode_i_u {
```

Comment:
Seems like some of this could be split out as a prep patch?

Reply:
Well, actually, it does tweak the code to work with some new naming in the new file, but its nice to merge those to have each commit build.

Reply:
The changes were proposed upstream today, so I think we can merge the whole thing soon.